go/vt/wrangler/resharder.go

```diff
@@ -47,6 +47,8 @@ type resharder struct {
 	targetMasters map[string]*topo.TabletInfo
 	vschema       *vschemapb.Keyspace
 	refStreams    map[string]*refStream
+	cell          string
```

Comment:
should document as "single cell or cellsAlias or comma-separated list of cells/cellsAliases"